__tests__/pages/aboutpage.test.tsx

```diff
@@ -0,0 +1,17 @@
+import About from "@/pages/about";
+import { render } from "@testing-library/react";
+import { axe } from "jest-axe";
+
+class ResizeObserver {
```

Comment:
Move this to the jest setup file so you don't have to redeclare this in other tests.